packages/serialization/json/kiota_serialization_json/json_serialization_writer.py

```diff
@@ -387,6 +387,10 @@ def get_serialized_content(self) -> bytes:
         stream = json_string.encode('utf-8')
         return stream
 
+    @staticmethod
```

Comment:
can we avoid expanding the API surface and look into an alternative here please?